### PR TITLE
Dex 668 ags delete fix

### PR DESCRIPTION
**Pull request by Atticus29**

### CODE REVIEW

File: src/models/assetGroupSighting/useDeleteAssetGroupSighting.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+import axios from 'axios';
+import { get } from 'lodash-es';
+import { formatError } from '../../utils/formatters';
+
+export default function useDeleteAssetGroupSighting() {
+  const [loading, setLoading] = useState(false);
+  const [error, setError] = useState(null);
+  const [success, setSuccess] = useState(false);
+
+  const deleteAssetGroupSighting = async assetGroupSightingId => {
```
Comment (brmscheiner) [inline src/models/assetGroupSighting/useDeleteAssetGroupSighting.js:11]: gah, i hate to throw this back again but we really need to be using react-query for all our new api hooks

**Review by brmscheiner — CHANGES_REQUESTED** [review]

File: src/models/assetGroupSighting/useDeleteAssetGroupSighting.js (hunk 1 above)
Comment (Atticus29) [reply src/models/assetGroupSighting/useDeleteAssetGroupSighting.js:11]: Do you want the old delete sighting to be updated to react-query as well?

Comment (brmscheiner) [reply src/models/assetGroupSighting/useDeleteAssetGroupSighting.js:11]: that would be nice, but not a requirement

File: src/models/assetGroupSighting/useDeleteAGSEncounter.js [hunk 2]
```diff
@@ -24,6 +24,8 @@ export default function useDeleteAGSEncounter() {
     ? mutation?.error.toJSON().message
     : null;
 
+  // const error = 'Test error 1';
+
```
Comment (brmscheiner) [inline src/models/assetGroupSighting/useDeleteAGSEncounter.js:28]: remove

File: src/pages/sighting/SightingCore.jsx [hunk 3]
```diff
@@ -86,9 +90,18 @@ export default function SightingCore({
...
-
+  useEffect(
+    () => {
+      const localDeleteErrorMessage = pending
+        ? deleteAssetGroupSightingError
+        : deleteSightingError;
+      setLocalDeleteError(localDeleteErrorMessage);
+    },
+    [deleteAssetGroupSightingError, deleteSightingError],
+  );
```
Comment (brmscheiner) [inline src/pages/sighting/SightingCore.jsx:104]: why not just `const localDeleteError = pending ? deleteAssetGroupSightingError : deleteSightingError;`. not fully seeing the point of useEffect and useState here

File: src/pages/sighting/encounters/Encounters.jsx [hunk 4]
```diff
@@ -79,6 +72,17 @@ export default function Encounters({
...
+
+  useEffect(
+    () => {
+      const localDeleteErrorMessage = pending
+        ? deleteAGSEncounterError
+        : deleteSightingEncounterError;
+      setLocalDeleteError(localDeleteErrorMessage);
+    },
+    [deleteAGSEncounterError, deleteSightingEncounterError],
+  );
```
Comment (brmscheiner) [inline src/pages/sighting/encounters/Encounters.jsx:85]: same question here

File: src/models/assetGroupSighting/useDeleteAGSEncounter.js (hunk 2 above)
Comment (Atticus29) [reply src/models/assetGroupSighting/useDeleteAGSEncounter.js:28]: Done.

File: src/components/Alert.jsx [hunk 5]
```diff
@@ -14,7 +14,7 @@ const Core = function({
 }) {
   return (
     <MuiAlert {...rest}>
-      {children}
+      {children},
```
Comment (brmscheiner) [inline src/components/Alert.jsx:17]: this looks phishy

**Review by brmscheiner — APPROVED** [review]

File: src/components/Alert.jsx (hunk 5 above)
Comment (Atticus29) [reply src/components/Alert.jsx:17]: Ah yeah. Vestige of something I was troubleshooting. Weird that the linters were fine with this.

**Pull request merged**